### DISCUSSION TITLE
why Empty class path is given???

### DISCUSSION THREAD

**Discussion by Nasiri2022** [soot-oss/SootUp#938] [Q&A]
It shows why Empty class path is given. to generate the call graph???

**Comment by swissiety** [comment]
Hello @Nasiri2022,
this seems to be your first discussion ever as github notes. Maybe you are so [polite](https://opensource.how/etiquette/) to elaborate on your question(s) *!!* !

TLDR: I dont know your SootUp setup but the classpath given to JavaClassPathAnalysisInputLocation does not exist on your filesystem.

**Reply by Nasiri2022** [reply]
Thank you for your answer
I did not explain because it was clear and your answer was completely correct
